b12x/integration/vllm/qsrt_plugin.py

```
@@ -0,0 +1,593 @@
+def register_b12x_qsrt() -> None:
+    """Register the ``b12x_qsrt`` quantization method in every vLLM process."""
+
+    global _registered, _CONFIG_CLS, logger
+    if _registered:
+        return
+    try:
+        from vllm.logger import init_logger
+
+        logger = init_logger("vllm.b12x_qsrt")
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)